src/index.js

```diff
@@ -21,6 +22,9 @@ const optionsDefault = {
   }
 };
 
+const COMMENT_START = /^<!--/g;
+const COMMENT_END = /-->$/g;
```

Comment:
))) I meant to make out the parts that are responsible for the comment

`<!--` and `-->`

Reply:
I didnt get this.
you meant to use with template literals for the regex test ?

Reply:
I initially thought of putting only these parts into a constant

Reply:
ohh, should I change that ?
we need those `^` and `$` as well.
 maybe these should be suffix with `PATTERN` ?

Reply:
yep